assets/data/modInjections/scenario/campaign/villain_cvawn.json

```diff
@@ -0,0 +1,11 @@
+{
+"modId": "wildermyth-drauven-pcs",
+"modInjections": [
+	{
+		"jsonPath": "scenarioInfo/rosterSlots/[2]/forbidLegacyHeroAspects",
+		"type": "stringList",
+		"addJson": { "class": "java.lang.String", "value": "" },
+		"addStrings": [ "drauven" ]
+	}
+]
```

Comment:
Let's also inject a `specificHumanBaseId` of `humanBase.001` for the other roster slots so that we can't get Drauven rolled up randomly once we get the new history-based Drauven out :)

<img width="149" height="16" alt="image" src="https://github.com/user-attachments/assets/ef2594e2-04e8-4f87-94f2-def1864565e2" />